docs/installation/community-operator.md

```diff
@@ -1 +1,38 @@
-# Kepler Community Operator on OpenShift
\ No newline at end of file
+# Kepler Community Operator on OpenShift
+
+## Install operator from Operator Hub
+
+Before you start make sure you have a OCP 4.13 cluster running.
+
+
+1. Go to Operator Hub. Search for `Kepler`. Click on `Install` 
+![](../fig/ocp_installation/operator_installation_ocp_1.png)
+
+2. Allow the operator to install
+![](../fig/ocp_installation/operator_installation_ocp_7.png)
+
+3. Create a Custom Resource for Kepler 
+![](../fig/ocp_installation/operator_installation_ocp_2.png)
+> Note: The current OCP console may display a JavaScript error (expected to be fixed in 4.13.5), but it doesn't affect the rest of the steps. Fixes are currently available on the 4.13.0-0.nightly-2023-07-08-165124 build of OCP console.
+
+
+## Installing the Grafana operator
+
```

Comment:
>Have a clone of kepler-operator repo since you need the hack files.
I am working towards injecting the API token via env variable so that the user do not have to clone and update the yaml. Instead they can then curl and execute the script.